### DISCUSSION TITLE
Im getting error while installing mongo driver

### DISCUSSION THREAD

**Discussion by Vigneshmunikannan** [jenssegers/laravel-mongodb#2789] [General]
pls have a look on this and give me solution

![image](https://github.com/mongodb/laravel-mongodb/assets/107925681/7dd59170-c551-4843-bc9c-7ccc84e382a2)

**Comment by bisht2050** [comment]
You seem to be missing installation of the MongoDB PHP Extension. You can find instructions for the same here - https://www.php.net/manual/en/mongodb.installation.php

And here's a quick start guide that should cover all installation and getting started steps for Laravel with MongoDB - https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/quick-start/download-and-install/

**Reply by Vigneshmunikannan** [reply]
i done that also but its not working can you pls help me out I'm doing this as my interview process I want to submit it by tomo can you help me now

**Reply by bisht2050** [reply]
Can you share what steps you undertook to install the extension?

**Reply by Vigneshmunikannan** [reply]
First I installed php, then I downloaded mongo php extracted it and i cutted and pasted it in php exe folder and i edited pho.ini added extension=mongodb but still didn't worked out for me pls help me out

**Reply by bisht2050** [reply]
Can you double check you are following exact steps, as they are written here - https://www.php.net/manual/en/mongodb.installation.windows.php
Specially around adding `extension=php_mongodb.dll`

**Reply by Vigneshmunikannan** [reply]
And i want you to let you know that Im doing this in vs code and i followed exact same method can we have g meet by 5:30
